hugr-py/rust/lib.rs

```diff
@@ -1,12 +1,15 @@
 //! Supporting Rust library for the hugr Python bindings.
 
+mod metadata;
 mod model;
 
 use pyo3::pymodule;
 
 #[pymodule]
 mod _hugr {
-
     #[pymodule_export]
     use super::model::model;
+
+    #[pymodule_export]
+    use super::metadata::metadata;
```

Comment:
Depending on whether you usually order these uses alphabetically, this one is in the wrong place (since "me..." < "mo...")